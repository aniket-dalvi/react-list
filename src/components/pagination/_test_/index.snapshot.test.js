import React from "react";
import render from "react-test-renderer";
import Pagination from "../../pagination";

describe("snapshot test for pagination", () => {
  test("snapshot: ", () => {
    const app = render
      .create(
        <Pagination
          maxElement={2}
          getList={(list) => console.log(list)}
          slot={5}
          activeCounter={1}
          currentPageNumber={(pageno) => console.log(pageno)}
        />
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
