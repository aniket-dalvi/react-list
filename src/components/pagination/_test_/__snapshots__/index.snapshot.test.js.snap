// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for pagination snapshot:  1`] = `null`;
